jupyterlab_bigquery/jupyterlab_bigquery/list_items_handler/create_handler.py

```diff
@@ -17,6 +17,22 @@ def _handle_exception(self, e):
     self.finish({"error": {"message": str(e)}})
 
 
+class Handlers():
+  """Singleton for handlers dictionary"""
+
+  _instance = {}
+
+  @classmethod
+  def get(cls):
+    if not cls._instance:
+      cls._instance = Handlers()
+    return cls._instance
+
+  @classmethod
+  def set_handler(cls, key, value):
+    cls._instance[key] = value
```

Comment:
This should be protected by calling `.get()` to make sure Handlers are instantiated.